packages/pwa/app/components/item-variant/index.jsx

```diff
@@ -9,30 +9,25 @@ import PropTypes from 'prop-types'
 
 /**
  * This component and associated context/hook provide a convenient wrapper
- * around a group of components used for rendering product variant details
- * from items in a customer basket or order.
+ * around a group of components used for rendering product variant details.
  */
 
-const CartItemVariantContext = createContext()
+const ItemVariantContext = createContext()
 
-export const useCartItemVariant = () => {
-    return useContext(CartItemVariantContext)
+export const useItemVariantContext = () => {
```

Comment:
I don't think this naming is correct.

```
const thing = useItemVariantContext() // typeof variant !== 'context' as the name of the function implies you are getting back 👎 
```

So, this is probably better:

```
const variant = useItemVariant() // typeof variant === 'variant'. 👍 
```

That being said.. what is a "ItemVariant" what modal does it relate to in the commerce rest api? I'm guessing that it originally was referring to a baskets "ProductItem" ?

Reply:
Good call, the `return useContext(CartItemVariantContext)` tricks me thinking that the returned value is a context.